consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -723,11 +722,19 @@ private void triggerSnapshotByCustomize() {
         continue;
       }
 
-      final long currentDirLength =
-          calcMap.computeIfAbsent(currentDir, MemorizedFileSizeCalc::new).getTotalFolderSize();
+      final long currentDirLength = monitor.updateAndGetDirectorySize(currentDir);
       final long triggerSnapshotFileSize = config.getImpl().getTriggerSnapshotFileSize();
 
       if (currentDirLength >= triggerSnapshotFileSize) {
+        final StringBuilder allFileNamesBuilder = new StringBuilder();
+        monitor.getFilesUnder(currentDir).forEach(p -> allFileNamesBuilder.append(p).append(","));
+        logger.info(
+            "{}: take snapshot for region {}, current dir size {}, files {}",
+            this,
+            raftGroupId,
+            currentDirLength,
+            allFileNamesBuilder);
```

Comment:
whether this field will be too long?

Reply:
I'll just print how many files contained.